storybook/storybook-docs/src/interactive-elements.stories.mdx

```diff
@@ -0,0 +1,66 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Designrichtlijnen/Interactieve elementen" />
+
+# Interactieve elementen
```

Comment:
In general, please uphold the distinction between:

1. Design choices implemented in our components. These can be descriptive “Knoppen zijn blauw (omdat …)”.
2. Guidelines for using our components. These should be written in normative voice: “Zorg ervoor dat…”

Implementers need to only understand the former, but act on the latter.

You already did this in the ‘Focus indicator’ section, but it is less clear in others.